cmd/dep/godep_importer_test.go

```diff
@@ -53,7 +236,7 @@ func TestGodepConfig_Import(t *testing.T) {
 		t.Fatal("Expected the lock to be generated")
 	}
 
-	goldenFile := "godep/expected_import_output.txt"
+	goldenFile := "godep/want_import_output.txt"
 	got := verboseOutput.String()
```

Comment:
Filename change 😬 seems to be causing the test to fail. No such file exists. I think it's fine to keep it `expected_import_output.txt`.